hnn_core/viz.py

```diff
@@ -101,9 +101,9 @@ def plot_extracellular(times, data, tmin=None, tmax=None, ax=None,
         The SciPy function :func:`~scipy.signal.decimate` is used, which
         recommends values <13. To achieve higher decimation factors, a list of
         ints can be provided. These are applied successively.
-    color : str | array of floats | ``matplotlib.colors.ListedColormap``
-        The colormap to use for plotting. The usual Matplotlib standard
-        colormap strings may be used (e.g., 'jetblue'). A color can also be
+    color : string | array of floats | ``matplotlib.colors.ListedColormap``
+        The color to use for plotting (optional). The usual Matplotlib standard
+        color strings may be used (e.g., 'b' for blue). A color can also be
```

Comment:
Were these supposed to be part of this PR as well?

Reply:
Oh, was the goal to move away from using a colormap? FYI, `str` is actually an object class, so it is an acceptable type for the `color` argument and doesn't need to by spelled out as `string`.